tests/components/discovergy/test_diagnostics.py

```diff
@@ -0,0 +1,78 @@
+"""Test Discovergy diagnostics."""
+from homeassistant.components.diagnostics import REDACTED
+from homeassistant.core import HomeAssistant
+
+from . import init_integration
+
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.typing import ClientSessionGenerator
+
+
+async def test_entry_diagnostics(
+    hass: HomeAssistant, hass_client: ClientSessionGenerator
+) -> None:
+    """Test config entry diagnostics."""
+    entry = await init_integration(hass)
+
+    result = await get_diagnostics_for_config_entry(hass, hass_client, entry)
+
+    assert result["entry"] == {
+        "entry_id": entry.entry_id,
+        "version": 1,
+        "domain": "discovergy",
+        "title": "user@example.org",
```

Comment:
Does the `title` and `unique_id` get set to a users email address under normal operation? If so, we should add those fields to the redacted list as well.